widgetry/src/canvas.rs

```diff
@@ -161,6 +161,10 @@ impl Canvas {
                     );
                 }
             }
+
+            if input.left_mouse_double_clicked() {
```

Comment:
This new behavior will happen anywhere normal canvas movement happens. I tried out edit mode, some simulation stuff, etc and I think it's all fine -- there's no existing controls we have that come close to double clicking, so there shoudn't be conflict